src/main/java/com/launchdarkly/client/RedisFeatureStore.java

```diff
@@ -278,10 +348,26 @@ public boolean initialized() {
    */
   public void close() throws IOException
   {
-    pool.destroy();
+    try {
+      if (executorService != null) {
+        executorService.shutdownNow();
+      }
+    } finally {
+      pool.destroy();
+    }
   }
 
-
+  /**
+   * Return the underlying Guava cache stats object.
+   *
+   * @return the cache statistics object.
+   */
+  public CacheStats getCacheStats() {
```

Comment:
This leaks implementation detail of the `RedisFeatureStore` so I added it so that we could discuss options. e.g. convert it to some sort a Map? Perhaps it's not that useful at all?